plugin/lspconfig.lua

```diff
@@ -0,0 +1,58 @@
+local api = vim.api
+local lspconfig = require 'lspconfig'
```

Comment:
Can `require 'lspconfig'` be inlined in all places where `lspconfig` is used? Then we don't force the module to be loaded.

Reply:
done